chains/manager.go

```diff
@@ -256,10 +258,7 @@ type manager struct {
 	chainCreatorShutdownCh chan struct{}
 	chainCreatorExited     sync.WaitGroup
 
-	subnetsLock sync.RWMutex
-	// Key: Subnet's ID
-	// Value: Subnet description
-	subnets map[ids.ID]subnets.Subnet
+	subnets *Subnets
```

Comment:
I think this needs to be removed (and it's usage replaced with `ManagerConfig.Subnets`

Reply:
Ah i fixed this on the chain manager PR but forgot to push the fix here as well. Will update.